simulation/agents.py

```diff
@@ -0,0 +1,74 @@
+import random
+
+import pandas as pd
+
+from abm import Agent
+from simulation.constants import (
+    HEATING_SYSTEM_FUEL,
+    HEATING_SYSTEM_LIFETIME_YEARS,
+    BuiltForm,
+    Epc,
+    HeatingFuel,
+    HeatingSystem,
+    OccupantType,
+    PropertyType,
+)
+from simulation.settings import HEAT_PUMP_AWARENESS
+
+
+class Household(Agent):
+    def __init__(
+        self,
+        location: str,
+        property_value: int,
+        floor_area_sqm: int,
+        off_gas_grid: bool,
+        construction_year_band: pd.Interval,
+        property_type: PropertyType,
+        built_form: BuiltForm,
+        heating_system: HeatingSystem,
+        epc: Epc,
+        occupant_type: OccupantType,
+        is_solid_wall: bool,
+        walls_energy_efficiency: int,
+        windows_energy_efficiency: int,
+        roof_energy_efficiency: int,
+        is_heat_pump_suitable_archetype: bool,
+    ):
+        # Property / tenure attributes
+        self.location = location
+        self.property_type = property_type
+        self.occupant_type = occupant_type
+        self.built_form = built_form
+        self.floor_area_sqm = floor_area_sqm
+        self.property_value = property_value
+        self.is_solid_wall = is_solid_wall
+        self.construction_year = self.sample_uniformly_from_interval(
```

Comment:
Do we need this staticmethod here, any more? Can we just use `random.randint(...)` since we want a full year and not a float, anyway? If so, we can simplify this a bit more and remove the `staticmethod` below.
N.B if we do want to estimate a construction datetime, then we should do that instead with a datetime object.